ukol08_Burjanova/src/main/java/cz/czechitas/java/database/Zamestnanec.java

```diff
@@ -1,19 +1,20 @@
 package cz.czechitas.java.database;
 
 public class Zamestnanec {
-    private int id_autor;
+
+    private Long id;
     private String jmeno;
     private String bydliste;
     private int plat;
-    private String datum_nastupu;
```

Comment:
Please, follow Java naming conventions (no underscore in names, names with lowercase first letter)